src/App.tsx

```diff
@@ -1,9 +1,136 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+const getUserbyId = (userId: number): User | undefined => {
+  return usersFromServer.find((user) => user.id === userId);
+};
+
+const todosWithUsers = todosFromServer.map((todo) => (
+  {
+    ...todo,
+    user: getUserbyId(todo.userId),
+  }
+));
+
+const getNewId = (todos: Todo[]) => {
```

Comment:
nit: Consider renaming the function getNewId to `getNewTodoId` to better represent its purpose. 🎯